cli/determined_cli/trial.py

```diff
@@ -205,6 +205,48 @@ def kill_trial(args: Namespace) -> None:
                             "of the log (default is all)",
                         ),
                     ),
+                    Arg(
+                        "--agent-id",
+                        dest="agent_ids",
+                        action="append",
+                        help="agents to show logs from (repeat for multiple values)",
+                    ),
+                    Arg(
+                        "--container-id",
+                        dest="container_ids",
+                        action="append",
+                        help="containers to show logs from (repeat for multiple values)",
+                    ),
+                    Arg(
+                        "--rank-id",
+                        dest="rank_ids",
+                        type=int,
+                        action="append",
+                        help="containers to show logs from (repeat for multiple values)",
+                    ),
+                    Arg(
+                        "--timestamp-before",
+                        help="show logs only from before (RFC 3339 format)",
+                    ),
+                    Arg(
+                        "--timestamp-after",
+                        help="show logs only from after (RFC 3339 format)",
+                    ),
+                    Arg(
+                        "--level",
+                        action="append",
+                        help="log levels to show logs from (repeat for multiple values)",
```

Comment:
blocking: Hmm, so because of the proto enum stuff, this ends up needing to take arguments that are _numbers_ rather than log levels, which is odd. Do you see a good way around that?

Also, I definitely should've thought of this before, but it's a little silly that you have to individually specify every log level to include, rather than the semantics being "the specified level or higher". Possibly this should be changed in the API, so not necessarily a blocker here. (Though I think I actually like the idea, for flexibility's sake, of the API doing what it does now and the interfaces doing the translation by default.)

Reply:
I guess this is the major downside of using the _not_ using the generated swagger client.. should figure that out sooner than later but for now I'll manually add a mapping.

Reply:
I blanked on this (shouldn't have responded while i was sick lol). The rest that gets generated is for a string enum not numbers, so it's a _lot_ more bareable than anticipated.. actually ends up not being too bad. Also changed it to log level above X.